packages/bpk-component-inset-banner/src/bpk-component-inset-banner-v2/src/BpkInsetBannerSponsored.tsx

```diff
@@ -0,0 +1,198 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useState } from 'react';
+
+import { surfaceHighlightDay } from '@skyscanner/bpk-foundations-web/tokens/base.es6';
+
+import BpkBottomSheet from '../../../../bpk-component-bottom-sheet';
+import ViewIcon from '../../../../bpk-component-icon/lg/view';
+import InfoIcon from '../../../../bpk-component-icon/sm/information-circle';
+import BpkImage from '../../../../bpk-component-image';
+import BpkText, { TEXT_STYLES } from '../../../../bpk-component-text/src/BpkText';
+import { cssModules } from '../../../../bpk-react-utils';
+
+import STYLES from './BpkInsetBannerSponsored.module.scss';
+
+const getClassName = cssModules(STYLES);
+
+export const VARIANT = {
+  onLight: 'on-light',
+  onDark: 'on-dark',
+};
+type callToActionType = {
+  text?: string;
+  bottomSheetContent: Array<{
+    title: string;
+    description: string;
+  }>;
+  bottomSheetTitle?: string;
+  buttonCloseLabel?: string;
+  buttonA11yLabel?: string;
+  bottomSheetLabel?: string;
+  bottomSheetId?: string;
+  bottomSheetWidth?: string;
+  bottomSheetMarginStart?: string;
+  bottomSheetMarginEnd?: string;
+  bottomSheetA11yLabel?: string;
+  labelTitle?: boolean;
+  closeBtnIcon?: boolean;
+  zIndexCustom?: number;
+}
+
+export type Props = {
+  accessibilityLabel?: string;
+  backgroundColor?: string;
+  callToAction?: callToActionType & {bottomSheetContent: callToActionType['bottomSheetContent']};
+  logo?: string;
+  subheadline?: string;
+  title?: string;
+  variant?: (typeof VARIANT)[keyof typeof VARIANT];
+  image?: {
+    src: string;
+    altText: string;
+    aspectRatio: number;
+  }
+};
+
+export const BpkInsetBannerSponsored = ({
```

Comment:
Hi Hiral, we get some concern from peers about naming the sponsored version might should be little business related and more commonly. How about rename it to `BpkInsetBannerPrompt`?

Reply:
Update: [Discuss](https://skyscanner.slack.com/archives/C0JHPDSSU/p1750670231598949?thread_ts=1748944143.139359&cid=C0JHPDSSU) with team, we can keep use Sponsored